head.go

```diff
@@ -367,6 +370,14 @@ func (h *Head) loadWAL(r *wal.Reader) error {
 	}
 	wg.Wait()
 
+	// Though we don't store tombstones anymore, we need this when user
+	// is migrating from the version where we used to store tombstones.
```

Comment:
This doesn't look right:

Imagine the following:
1. Delete some series from the head.
1. Stop Prometheus before the head is persisted into a block
1. Start Prometheus.

At this point Prometheus reads the wal so all data that was deleted in the head would be loaded again from the wal.

Reply:
Didn't think of that. I had removed writing of stones into the WAL after `Delete()`, I will bring it back.